ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -56,15 +59,51 @@ const PostBreakingNewsLabel = ({
   ) : null;
 };
 
+const TimeStamp = ({ curated }: { curated: string }) => {
```

Comment:
Would it make sense to refactor this into a separate file within the Post directory?